Bottom: 2a01297827efe9658a568bb83d6813b9de9d1edf
Top:    29219208e742a77eee2e11c8c2dbd7c9bed26e80
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:51:28 +0000

Refresh of subtract-vias-from

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index f880b52..7347ce3 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1306,6 +1306,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
